C++/Logger/SyncFileSink.h

```diff
@@ -1,25 +0,0 @@
-#ifndef SYNC_FILE_SINK_H
-#define SYNC_FILE_SINK_H
-
-#include "Sink.h"
-#include <mutex>
```

Comment:
What about something like this?
```
// --------------------------------------------------
#include "StdOutSink.h"
// --------------------------------------------------
#include <iostream>
// --------------------------------------------------
```
We do this at work and I kind of like it, they also do it between each function definition.